OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/pdf/PDFDisplay.java

```diff
@@ -36,9 +36,13 @@ public JPanel drawPreviewOnJPanel() {
         Image generatedImage = PDFPreviewBuilder.buildPDFPreview(selectedPage, viewZoom);
         
         if (generatedImage != null) {
-            ImageIcon icon = new ImageIcon(generatedImage);
-            JLabel picLabel = new JLabel(icon);
-            pdfImagePanel.add(picLabel);
+            try {
+                ImageIcon icon = new ImageIcon(generatedImage);
+                JLabel picLabel = new JLabel(icon);
+                pdfImagePanel.add(picLabel);
+            } catch (OutOfMemoryError oome) {
```

Comment:
I'll check it out.   As far as the error goes, check me on this, but doesn't the Image class throw an error because of the image file and we catch the error here and disregard it.  The image never gets made and the image returns a null which displays as a blank screen on the preview.  We can't make it work with the current memory limit but this way it doesn't present an error the preview just goes away.  Btw with the image then cleared the program usually has enough memory to display the next bump.  With my system it's currently crashing at 190 but then can display 200 without a problem.  Other than increasing the memory how else would we deal with this?  This is the first serious java project I've been on so just trying to understand.